ray-operator/config/samples/ray-cluster-fluentbit-sidecar.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: fluentbit-config
+data:
+  fluent-bit.conf: |
+    [INPUT]
+        Name tail
+        Path /tmp/ray/session_latest/logs/*
+        Tag ray
+        Path_Key true
+        Refresh_Interval 5
+    [FILTER]
+        Name modify
+        Match ray
+        Add POD_LABELS ${POD_LABELS}
+    [OUTPUT]
+        Name loki
+        Match *
+        Host loki-gateway
+        Port 80
+        Labels RayCluster=${POD_LABELS}
+        tenant_id test
+---
+# RayCluster CR with a FluentBit sidecar
+apiVersion: ray.io/v1alpha1
+kind: RayCluster
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: raycluster-fluentbit-sidecar-logs
+spec:
+  rayVersion: '2.9.0'
+  headGroupSpec:
+    rayStartParams:
+      dashboard-host: '0.0.0.0'
```

Comment:
Remove `dashboard-host: '0.0.0.0'`, as KubeRay has already set it as the default value, and use `rayStartParams: {}` instead.